macros/sql/pivot.sql

```diff
@@ -0,0 +1,62 @@
+{#
+Pivot values from rows to columns.
+
+Example:
+
+    Input:
+
+    | size | color |
+    |------+-------|
+    | S    | red   |
+    | S    | blue  |
+    | S    | red   |
+    | M    | red   |
+
+    select
+      size,
+      {{ pivot('size', ['red', 'blue']) }}
```

Comment:
we don't do a great job of documenting this, but macros in this package need to be namespaced to `dbt_utils`, ie. 

```
{{ dbt_utils.pivot(....) }}
```